.env

```diff
@@ -1,11 +1,11 @@
 # Django settings
 SECRET_KEY=dUmMy_s3cR3t_k3y
-#DB_ENGINE=mysql
-#DB_NAME=yaksh
-#DB_USER=root
-#DB_PASSWORD=root
+DB_ENGINE=mysql
```

Comment:
Don't rename .samplenv to .env this is not required it will be done in the production.